### PR TITLE
Removed color and imageType from the visualFeature enumeration in the V2 SDK.

### PR DESCRIPTION
**Pull request by jennifermarsman**
Removed color and imageType from the visualFeature enumeration in the V2 SDK.  These weren't referenced in the examples.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [X] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [X] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by jennifermarsman** [comment]
Adding @Careyjmac as a SME as well.

**Comment by Careyjmac** [comment]
Are we only changing this in the preview version?  We need to change this in the GA version as well (even though its unfortunately breaking and will be yet another major version number increase to do so)

**Comment by brjohnstmsft** [comment]
@Careyjmac As of now, the new SDKs under development will only be using the Track 2 2019-05-06-Preview specs. All other Swaggers should be frozen from here on out. We don't plan to ship an update of the existing Track 1 .NET SDK unless there is an urgent need (e.g. -- security flaw that requires a hotfix).

**Comment by Careyjmac** [comment]
> @Careyjmac As of now, the new SDKs under development will only be using the Track 2 2019-05-06-Preview specs. All other Swaggers should be frozen from here on out. We don't plan to ship an update of the existing Track 1 .NET SDK unless there is an urgent need (e.g. -- security flaw that requires a hotfix).

Gotcha.  So we are no longer going to have separate SDK versions (i.e. version# and version#-preview) where the preview on contains the features that are only in the preview api (things such as knowledge store, annotation cache, etc.)?  That's fine, we'll just need some other way to clearly define which features are preview only no?  Or is the guidance going forward to only ever include features that are considered GA?

**Comment by brjohnstmsft** [comment]
@Careyjmac That is still an unsolved problem as of now. The new [API guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-versioning) say that a single library needs to be able to support multiple API versions in parallel, but there will be challenges for us with that approach. If it works though, the GA SDK will likely support both 2019-05-06-Preview and the next (as-yet-unnamed) GA API version together in the same library.

File: specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json [hunk 1]
```diff
@@ -5514,10 +5514,8 @@
         "adult",
         "brands",
         "categories",
-        "color",
```
Comment (brjohnstmsft) [inline specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json]: @jennifermarsman I thought the whole type was going away -- or did I misunderstand the plan?

**Comment by brjohnstmsft** [comment]
@jennifermarsman You can ignore the `MISSING_REQUIRED_PARAMETER` model validation errors. It's a known issue with the model validator not being able to parse our path format (because they use OData syntax). The team who owns the validator is aware of it and are working on a fix. When it comes time to merge this, we'll need to find the right person with permissions to merge despite validation errors.

File: specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json (hunk 1 above)
Comment (brjohnstmsft) [reply specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json]: Nevermind, I read more closely. This looks fine.

**Review by bitsforthought — APPROVED** [review]

**Comment by brjohnstmsft** [comment]
FYI @jhendrixMSFT -- This change is for a feature we're deprecating and therefore don't want in the new Track 2 SDKs.

@mayurid -- We'll need your help again to merge this until the model validator bug is fixed.

**Review by Careyjmac — APPROVED** [review]

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**